src/api/index.test.js

```diff
@@ -235,7 +340,17 @@ it('deletePipelineRun', () => {
   const name = 'foo';
   const data = { fake: 'pipelineRun' };
   fetchMock.delete(`end:${name}`, data);
-  return deletePipelineRun({ name }).then(pipelineRun => {
+  return index.deletePipelineRun({ name }).then(pipelineRun => {
+    expect(pipelineRun).toEqual(data);
+    fetchMock.restore();
+  });
+});
+
+it('deletePipelineRun', () => {
+  const name = 'foo';
+  const data = { fake: 'pipelineRun' };
+  fetchMock.delete(`end:${name}`, data);
```

Comment:
Same as above wrt the end

Reply:
This is someones PR, i just added the `index.` before because had to add the index for jest to work